mypy/checker.py

```diff
@@ -4001,6 +4002,15 @@ def find_isinstance_check_helper(self, node: Expression) -> Tuple[TypeMap, TypeM
                 if literal(expr) == LITERAL_TYPE:
                     vartype = type_map[expr]
                     return self.conditional_callable_type_map(expr, vartype)
+            else:
+                if (isinstance(node.callee, RefExpr)
+                        and isinstance(node.callee.node, SYMBOL_FUNCBASE_TYPES)
+                        and isinstance(node.callee.node.type, CallableType)
+                        and node.callee.node.type.type_guard is not None):
+                    if len(node.args) < 1:  # TODO: Is this an error?
```

Comment:
This seems fine to me, it's similar to `if isinstance()` / you have a TODO in your tests to error at the definition site.